recipe/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "2.12.0" %}
+{% set name = "tensorflow-text" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/text
+  git_rev: v{{ version }}
+  patches:
+    - 0001-fix-build.sh
+
+build:
+  number: 0
+  # skipping s390x and win-64 since we don't have tf for it
+  skip: True  #[s390x or win]
```

```suggestion
  skip: True  #[s390x or win]
   script:
    ./oss_scripts/run_build.sh
    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

But then you need to do something with this error on MacOS:
```
Installing tensorflow.
+++ arch
++ [[ xarm64 == x\a\r\m\6\4 ]]
++ pip install tensorflow-macos==2.9.0
ERROR: Could not find a version that satisfies the requirement tensorflow-macos==2.9.0 (from versions: none)
ERROR: No matching distribution found for tensorflow-macos==2.9.0
```